autoscale-scaler-kubernetes/src/main/java/com/github/autoscaler/scaler/kubernetes/K8sServiceScaler.java

```diff
@@ -133,7 +133,7 @@ public InstanceInfo getInstanceInfo(final String resourceId) throws ScalerExcept
     @Override
     public HealthResult healthCheck()
     {
-        return K8sHealthCheck.healthCheck();
+        return K8sHealthCheck.healthCheck(this.config);
```

Comment:
Just `config` is fine.  The only place I would use `this.` is in the constructor, or in another method if there was some ambiguity.

```suggestion
        return K8sHealthCheck.healthCheck(config);
```